lib/checkWin.js

```diff
@@ -34,31 +34,31 @@ const checkMappedGridForWin = (array) => {
 };
 
 const checkForDraw = (store) => {
-  if (store.choices.length === 9 && store.winner === false) {
+  if (store.choices.length === 9 && !store.winner) {
     console.log("You drew!");
     return true;
   } else {
     return false;
   }
 };
 
-const checkForWin = (store) => {
+const checkForWin = async (store) => {
   let winner = checkMappedGridForWin(mapGridValues(store));
   if (winner === store.userTeam) {
     logGrid(store);
     console.log("You won!");
-    return playAgain();
+    return await playAgain(store);
   } else if (winner === store.cpuTeam) {
     logGrid(store);
     console.log("CPU won!");
-    return playAgain();
+    return await playAgain(store);
   }
 };
 
-const checkForGameOver = (store) => {
+const checkForGameOver = async (store) => {
   checkForWin(store);
```

Comment:
when you change a function into an async function, it's good practice to hunt for all usages of it and `await` them.

Reply:
I *think* what is happening

checkForWin(store) -> playAgain(store) -> resetStore(store) -> store.winner = false; ...play a new game...

(I don't think that you ever wet store.winner to true)

and in parallel:

userTurn(store) ->
    checkForGameOver(store) ->
        checkForDraw(store) -> false
    <- false
    ... keep on playing and realise that someone has already won
    ... ask if they want to play another game

I would actually like to draw this on a whiteboard with you before we go on holiday, but the short answer is that if you `await` here then all of the "in parallel" things go away (checkForWin() is recursive, and never resolves), so the bug goes away.